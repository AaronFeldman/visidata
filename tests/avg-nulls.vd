sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	Item		select-equal-cell		,	
sample		0	unselect-row		u	
sample	Units		delete-cells		gzd	
sample	Units		type-int		#	
	sample	error_is_null	set-option	True		
sample	Units		aggregate-col	avg	+	
sample	Item		freq-col		F	
