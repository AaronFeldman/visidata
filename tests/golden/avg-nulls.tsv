Item	count	avg_Units
Binder	15	48.13
Pencil	13	95.00
Pen Set	7	56.43
Pen	5	55.60
Desk	3	3.33
